docs/user/openstack/install_upi.md

```diff
@@ -264,16 +264,30 @@ $ tree
 └── install-config.yaml
 ```
 
-### Fix the Node Subnet
+### Configure the machineNetwork.CIDR apiVIP and ingressVIP.
+The `machineNetwork` represents the OpenStack network which will be used to connect all the OpenShift cluster nodes.
+The `machineNetwork.CIDR` defines the IP range, in CIDR notation, from which the installer will choose what IP addresses
+to assign the nodes.  The `apiVIP` and `ingressVIP` are the IP addresses the installer will assign to the cluster API and
+ingress VIPs, respectively.
+In the previous steps, the installer added default values for the `machineNetwork.CIDR`, and then it picked the
+5th and 7th IP addresses from that range to assign to `apiVIP` and `ingressVIP`.
+`machineNetwork.CIDR` needs to match the IP range specified by `os_subnet_range` in the inventory.yaml file.
 
-The installer added a default IP range for the OpenShift nodes. It must match the range for the Neutron subnet you'll create later on.
+When the installer creates the manifest files from an existing `install-config.yaml` file, it validates that the
+`apiVIP` and `ingressVIP` fall within the IP range specified by `machineNetwork.CIDR`. If they do not, it errors out.
+If you change the value of `machineNetwork.CIDR` you must make sure the `apiVIP` and `ingresVIP` values still fall within
+the new range. There are two options for setting the `apiVIP` and `ingressVIP`. If you know the values you want to use,
+you can specify them in the `install-config.yaml` file. If you want the installer to pick the 5th and 7th IP addresses in the
+new range, you need to remove the `apiVIP` and `ingressVIP` entries from the `install-config.yaml` file.
 
-We're going to use a custom subnet to illustrate how that can be done.
+To illustrate the process, we will use '192.0.2.0/24' as an example. It defines a usable IP range from
+192.0.2.1 to 192.0.2.254. There are some IP addresses that should be avoided because they are usually taken up or
+reserved. For example, the first address (.1) is could be assigned to a router. The DHCP and DNS servers will use a few
```

Comment:
> the first address (.1) is could be assigned to a router

There's one extra verb that slipped in. Maybe you could say `the first address (.1) is usually assigned to a router`

Reply:
how that get in there???
will correct.